radis/lbl/factory.py

```diff
@@ -1810,17 +1818,33 @@ def fit_spectrum(
         plot: bool
             if True, plot spectra as they are computed; and plot the convergence of
             the residual.
-        maxiter: int
-            max number of iteration, default 300
+        solver_options: dict
+            parameters forwarded to the solver. More info in `~scipy.optimize.minimize`
+            Example::
+
+                {"maxiter": (int)  max number of iteration default ``300``,
+                 }
+        kwargs: dict
+            forwarded to :py:func:`~radis.tools.fitting.fit_spectrum`
 
         Returns
         -------
-        s_best
+        s_best: Spectrum
+            best spectrum
+        res: OptimizeResults
+            output of `~scipy.optimize.minimize`
+
+        Examples
+        --------
+
+        .. minigallery:: radis.lbl.factory.fit_spectrum
```

Comment:
yes ! It will only be generated [on Readthedocs](https://radis.readthedocs.io/en/latest/gen_modules/radis.html#radis.SpectrumFactory.fit_spectrum) (once merged)